### ISSUE TITLE
[REQ] Maintain page position when Refresh button pressed

### ISSUE THREAD

**Issue by opav** [gorhill/uBlock#988]
In both uBlock and uMatrix, when I click on the refresh button inside, the page will reload and position is reset to the top of page.  

If the position on page is maintained, it will ease when switching around options to see effect of changes especially on long pages.

**Comment by gorhill** [comment]
The position is maintained with Chromium, the issue is specific to Firefox.

**Comment by opav** [comment]
If I use Firefox's own refresh function, the page position is maintained after refresh.

**Comment by gorhill** [comment]
That does not make it less of a Firefox issue. Let me rephrase: The position is maintainer with Chromium when refreshing through the API. The position is not maintained with Firefox when refreshing through the API.
